osquery/dispatcher/dispatcher.h

```diff
@@ -28,9 +28,9 @@ namespace osquery {
 class Status;
 class Dispatcher;
 
-class InterruptableRunnable {
+class interruptibleRunnable {
```

Comment:
class names should be Pascal Case
```suggestion
class InterruptibleRunnable {
```